### PR TITLE
feat: add customization for code block

### PR DESCRIPTION
**Pull request by ivamuno**

### CODE REVIEW

File: src/common-elements/PrismDiv.tsx [hunk 1]
```diff
@@ -48,7 +48,9 @@ export const PrismDiv = styled.div`
     color: hsl(30, 20%, 50%);
   }
 
+  .ellipsis,
```
Comment (gomete) [inline src/common-elements/PrismDiv.tsx:51]: What was the previous style for `.ellipsis`.

File: src/theme.ts [hunk 2]
```diff
@@ -224,12 +224,19 @@ const defaultTheme: ThemeInterface = {
...
-    backgroundColor: '#263238',
+    backgroundColor: '#3d434a', //'#263238',
     width: '40%',
-    textColor: '#ffffff',
+    textColor: '#ffffff'
   },
   codeBlock: {
     backgroundColor: ({ rightPanel }) => darken(0.1, rightPanel.backgroundColor),
+    colors: {
+      symbol: ({rightPanel}) => rightPanel.textColor,
```
Comment (gomete) [inline src/theme.ts:245]: Default? It was used in two different places, one it was `#fff`, the other wasn't set.

File: src/theme.ts [hunk 3]
```diff
@@ -224,12 +224,19 @@ const defaultTheme: ThemeInterface = {
...
+      property: ({codeBlock}) => codeBlock.colors.symbol,
```
Comment (gomete) [inline src/theme.ts:246]: Default `white`.

File: src/common-elements/PrismDiv.tsx (hunk 1 above)
Comment (ivamuno) [reply src/common-elements/PrismDiv.tsx:51]: It inherited from rightPanel.textColor 😮, I prefer to group all delimitators (or similar) under the same style within the codeBlock property.

File: src/theme.ts (hunk 3 above)
Comment (ivamuno) [reply src/theme.ts:246]: To keep backward compatibility it has to inherit from rightPanel.textColor 😅.

File: src/theme.ts [hunk 4]
```diff
@@ -224,21 +230,32 @@ const defaultTheme: ThemeInterface = {
     gutter: '2px',
   },
   rightPanel: {
-    backgroundColor: '#263238',
+    backgroundColor: '#3d434a', //'#263238',
```
Comment (gomete) [inline src/theme.ts]: Remove before merge.

**Pull request merged**